lib/docker.js

```diff
@@ -149,13 +161,13 @@ function createDockerFile() {
     }
 
     if (opts.deploy) {
-        contents += 'CMD /usr/bin/npm install --production && /usr/bin/npm install heapdump';
+        contents += 'CMD npm install --production && npm install heapdump';
```

Comment:
this needs to be:

```js
contents += 'CMD . $NVM_DIR/nvm.sh && nvm use ' + ndoeVersion + ' && npm install --production && npm install heapdump';
```

Otherwise, the system `node` binary is used.